[2023-10-17T11:19:12.064+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-13T00:00:00+00:00 [queued]>
[2023-10-17T11:19:12.072+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-13T00:00:00+00:00 [queued]>
[2023-10-17T11:19:12.072+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-17T11:19:12.092+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-03-13 00:00:00+00:00
[2023-10-17T11:19:12.099+0000] {standard_task_runner.py:57} INFO - Started process 569 to run task
[2023-10-17T11:19:12.102+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-03-13T00:00:00+00:00', '--job-id', '76', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmph96f4jnb']
[2023-10-17T11:19:12.103+0000] {standard_task_runner.py:85} INFO - Job 76: Subtask waiting_for_data
[2023-10-17T11:19:12.144+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-13T00:00:00+00:00 [running]> on host 238c1d4ff767
[2023-10-17T11:19:12.224+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-03-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-03-13T00:00:00+00:00'
[2023-10-17T11:19:12.234+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-17T11:19:12.235+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-17T11:19:12.235+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-17T11:19:12.235+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-17T11:19:12.245+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210313T000000, start_date=20231017T111912, end_date=20231017T111912
[2023-10-17T11:19:12.274+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-17T11:19:12.293+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T10:42:38.968+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-13T00:00:00+00:00 [queued]>
[2023-10-18T10:42:38.976+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-13T00:00:00+00:00 [queued]>
[2023-10-18T10:42:38.977+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T10:42:38.985+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-03-13 00:00:00+00:00
[2023-10-18T10:42:38.990+0000] {standard_task_runner.py:57} INFO - Started process 1530 to run task
[2023-10-18T10:42:38.992+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-03-13T00:00:00+00:00', '--job-id', '75', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpt85wvo5_']
[2023-10-18T10:42:38.993+0000] {standard_task_runner.py:85} INFO - Job 75: Subtask waiting_for_data
[2023-10-18T10:42:39.026+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-13T00:00:00+00:00 [running]> on host 2473b23ba069
[2023-10-18T10:42:39.078+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-03-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-03-13T00:00:00+00:00'
[2023-10-18T10:42:39.090+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 257, in execute
    raise e
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 239, in execute
    poke_return = self.poke(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/filesystem.py", line 61, in poke
    hook = FSHook(self.fs_conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/filesystem.py", line 38, in __init__
    conn = self.get_connection(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/base.py", line 72, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/connection.py", line 477, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `fs_default` isn't defined
[2023-10-18T10:42:39.093+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210313T000000, start_date=20231018T104238, end_date=20231018T104239
[2023-10-18T10:42:39.100+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 75 for task waiting_for_data (The conn_id `fs_default` isn't defined; 1530)
[2023-10-18T10:42:39.124+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-10-18T10:42:39.151+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T11:17:11.010+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-13T00:00:00+00:00 [queued]>
[2023-10-18T11:17:11.027+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-13T00:00:00+00:00 [queued]>
[2023-10-18T11:17:11.028+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T11:17:11.046+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-03-13 00:00:00+00:00
[2023-10-18T11:17:11.058+0000] {standard_task_runner.py:57} INFO - Started process 338 to run task
[2023-10-18T11:17:11.065+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-03-13T00:00:00+00:00', '--job-id', '103', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmp84inx1j_']
[2023-10-18T11:17:11.067+0000] {standard_task_runner.py:85} INFO - Job 103: Subtask waiting_for_data
[2023-10-18T11:17:11.122+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-13T00:00:00+00:00 [running]> on host 0e68aa52dfc7
[2023-10-18T11:17:11.243+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-03-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-03-13T00:00:00+00:00'
[2023-10-18T11:17:11.265+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-18T11:17:11.266+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-18T11:17:11.266+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-18T11:17:11.266+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-18T11:17:11.290+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210313T000000, start_date=20231018T111711, end_date=20231018T111711
[2023-10-18T11:17:11.356+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-18T11:17:11.396+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-10-19T12:11:49.905+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-13T00:00:00+00:00 [queued]>
[2023-10-19T12:11:49.915+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-13T00:00:00+00:00 [queued]>
[2023-10-19T12:11:49.915+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-19T12:11:49.928+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-03-13 00:00:00+00:00
[2023-10-19T12:11:49.936+0000] {standard_task_runner.py:57} INFO - Started process 391 to run task
[2023-10-19T12:11:49.938+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-03-13T00:00:00+00:00', '--job-id', '103', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpiylj5i0x']
[2023-10-19T12:11:49.939+0000] {standard_task_runner.py:85} INFO - Job 103: Subtask waiting_for_data
[2023-10-19T12:11:49.982+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-13T00:00:00+00:00 [running]> on host 6f5df7fea6fd
[2023-10-19T12:11:50.061+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-03-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-03-13T00:00:00+00:00'
[2023-10-19T12:11:50.090+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-19T12:11:50.090+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-19T12:11:50.091+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-19T12:11:50.091+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-19T12:11:50.102+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210313T000000, start_date=20231019T121149, end_date=20231019T121150
[2023-10-19T12:11:50.152+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-19T12:11:50.177+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
